CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -364,6 +390,9 @@ public function postProcess() {
     $statusMsg = ts('Event registration information for %1 has been updated.', array(1 => $displayName));
     $statusMsg .= ' ' . ts('A confirmation email has been sent to %1.', array(1 => $email));
     CRM_Core_Session::setStatus($statusMsg, ts('Registration Transferred'), 'success');
+    if (!empty($this->_isBackoffice)) {
```

Comment:
you can just use if ($this->isBackOffice) - since it's defined on the form it won't e-notice